cou/apps/base.py

```diff
@@ -579,28 +591,31 @@ def _get_charmhub_migration_step(self, target: OpenStackRelease) -> PreUpgradeSt
             ),
         )
 
-    def _get_change_to_openstack_channels_step(self, target: OpenStackRelease) -> PreUpgradeStep:
-        """Get the step for changing to an OpenStack channel.
+    def _get_change_channel_possible_downgrade_step(
+        self, target: OpenStackRelease, channel: str
+    ) -> tuple[str, Coroutine]:
```

Comment:
Why not returning `PreUpgradeStep`, since we are returning only parameter to create it.